yt/fields/field_aliases.py

```diff
@@ -116,11 +116,13 @@
     ("VorticityRPGrowthTimescale", "vorticity_radiation_pressure_growth_timescale"),
     ("DiskAngle", "theta"),
     ("Height", "height"),
-    ("HI density", "H_density"),
+    ("HI density", "H_p0_density"),
     ("HII density", "H_p1_density"),
-    ("HeI density", "He_density"),
+    ("HeI density", "He_p0_density"),
     ("HeII density", "He_p1_density"),
     ("HeIII density", "He_p2_density"),
+    ("H_density", "H_p0_density"),
+    ("He_density", "He_p0_density"),
```

Comment:
I think we need to be careful with these two. I think some frontends have aliases `X_number_density` to `X_nuclei_density`, which would be inconsistent with this. The `X_[number_]density` fields have changed meaning a few times. They were once the primary field for neutral ions, then the alias, and now even other things (like nuclei density) in other places. My preference would be to retire them altogether and update YTEP-0003 to reflect that.

Reply:
@brittonsmith Do you think that should happen here before this is merged?

Reply:
Yes, I would prefer that I think.